diff --git a/node_modules/@remix-run/router/dist/utils.d.ts b/node_modules/@remix-run/router/dist/utils.d.ts
index 3944262..9e52b4d 100644
--- a/node_modules/@remix-run/router/dist/utils.d.ts
+++ b/node_modules/@remix-run/router/dist/utils.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="react" />
 import type { Location, Path, To } from "./history";
 import { DataResult, DataRouteMatch } from "./router";
 export declare type FormMethod = "get" | "post" | "put" | "patch" | "delete";
@@ -74,13 +73,13 @@ export interface ShouldRevalidateFunction {
 export interface RouteObject {
     caseSensitive?: boolean;
     children?: RouteObject[];
-    element?: React.ReactNode;
+    element?: any;
     index?: boolean;
     path?: string;
     id?: string;
     loader?: LoaderFunction;
     action?: ActionFunction;
-    errorElement?: React.ReactNode;
+    errorElement?: any;
     shouldRevalidate?: ShouldRevalidateFunction;
     handle?: any;
 }
